templates/rhpam73-authoring-ha.yaml

```diff
@@ -1269,21 +1299,174 @@ objects:
   metadata:
     labels:
       application: ${APPLICATION_NAME}
-      service: ${APPLICATION_NAME}-amq
-    name: ${APPLICATION_NAME}-amq
+      service: ${APPLICATION_NAME}-amq-scaledown-controller
+    name: ${APPLICATION_NAME}-amq-scaledown-controller
     annotations:
+      deployment.kubernetes.io/revision: "1"
       template.alpha.openshift.io/wait-for-ready: "true"
   spec:
     replicas: 1
     selector:
-      deploymentConfig: ${APPLICATION_NAME}-amq
+      deploymentConfig: ${APPLICATION_NAME}-amq-scaledown-controller
     strategy:
       rollingParams:
         maxSurge: 0
       type: Rolling
     template:
       metadata:
         labels:
+          application: ${APPLICATION_NAME}
+          deploymentConfig: ${APPLICATION_NAME}-amq-scaledown-controller
+        name: ${APPLICATION_NAME}-amq-scaledown-controller
+      spec:
+        serviceAccountName: ${APPLICATION_NAME}-amq-scaledown-controller-sa
+        containers:
+        - args:
+          - --localOnly
+          - --v
+          - "3"
+          image: ${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAME}
+          imagePullPolicy: IfNotPresent
+          name: ${APPLICATION_NAME}-amq
+        terminationGracePeriodSeconds: 30
+    triggers:
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - ${APPLICATION_NAME}-amq
+        from:
+          kind: ImageStreamTag
+          namespace: "${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAMESPACE}"
+          name: "${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAME}:${AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_TAG}"
+      type: ImageChange
+    - type: ConfigChange
+- kind: StatefulSet
+  apiVersion: apps/v1
+  metadata:
+    annotations:
+      alpha.image.policy.openshift.io/resolve-names: '*'
+      statefulsets.kubernetes.io/drainer-pod-template: |
```

Comment:
Can this embedded JSON instead be embedded YAML? It's weird to see 2 file formats in the same file. If it's not possible, then okay.